docs/redoc/master/openapi.json

```diff
@@ -7253,7 +7253,7 @@
           }
         }
       },
-      "StrictModeConfig": {
+      "StrictModeConfigOutput": {
```

Comment:
This means generated clients will get a different type name, and that might need migration by users. Is there no other way to achieve the same without splitting into two types?

Reply:
To recap about the blast radius of the change:
- this is a change only for the REST clients (Python & js)
- this is an output type that users never construct

The content of the objects are identical, so it will be observable only by users explicitly type checking against the `StrictModeConfig` type which seems unlikely in Python & js.

I do not see another way of pleasing the current Pydantic setup without changing it.

Reply:
> The content of the objects are identical

In that case I don't see why we need a different object at all.

> I do not see another way of pleasing the current Pydantic setup without changing it.

What do you think about my above comment? Don't you agree pydantic should ignore unknown fields? If it doesn't, it'll be very hard to add new fields (across the API) in the future too.

Reply:
> In that case I don't see why we need a different object at all.

For backward compatibility purposes. 

> I'd argue that pydantic should ignore unknown field instead.

This is undesirable. We want validation of inputs. I don't remember the exact case why, but it was strong enough to go through this separation of input-output in the first place